docs/changelog2.rst

```diff
@@ -0,0 +1,241 @@
+  convert to an array of coordinates, use `.coords` instead
+  (`np.asarray(geom.coords)`).
+* Consistent creation of empty geometries (for exampe ``Polygon()`` now
+  actually creates an empty Polygon instead of an empty geometry collection).
+* The following attributes and methods on the Geometry classes were
+  deprecated and are now removed:
+  * ``array_interface()`` and ``ctypes``
+  * ``asShape()``, and the adapters classes to create geometry-like proxy
+    objects (use ``shape()`` instead).
+  * ``empty()`` method
+
+Breaking API changes
+^^^^^^^^^^^^^^^^^^^^
+
+Some additional backwards incompatible API changes were included in Shapely
+2.0 that were not yet deprecated in Shapely 1.8:
+
+* The default of the ``preserve_topology`` keyword of ``simplify()`` changed
+  to True (#1392).
+* A ``GeometryCollection`` that consists of all empty sub-geometries now
+  returns those empty geometries in ``.geoms`` (instead of returning an empty
+  list) (#1420).
+* The unused ``shape_factory()`` method and ``HeterogeneousGeometrySequence``
+  class are removed (#1421).
+* The undocumted ``__geom__`` attribute is removed. To access the raw GEOS pointer,
+  the ``_geom`` attribute is still present (1417).
+
+New features
+^^^^^^^^^^^^
+
+More informative repr with (truncated) WKT
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+Support for fixed precision model for geometries and in overlay functions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* `set_precision` to conform a geometry to a certain grid size, and this will then also be used by subsequent overlay methods
+* `grid_size` keyword in overlay methods (`intersection`, `union`, `difference`, etc) 
+
+
+* Addition of ``get_precision`` to get precision of a geometry and ``set_precision``
+  to set the precision of a geometry (may round and reduce coordinates) (#257)
+* Addition of ``grid_size`` parameter to specify fixed-precision grid for ``difference``,
+  ``intersection``, ``symmetric_difference``, ``union``, and ``union_all`` operations for
+  GEOS >= 3.9 (#276)
+
+Releasing the GIL
+~~~~~~~~~~~~~~~~~
+
+* Release the GIL to allow for multithreading in functions that do not
+  create geometries (#144) and in the STRtree ``query_bulk()`` method (#174)
+* Released the GIL in all geometry creation functions (#310, #326).
+
+STRtree improvements
+~~~~~~~~~~~~~~~~~~~~
+
+* Specifying a `predicate` in the STRtree queries 
+* Bulk queries
+
+
+* STRtree improvements for spatial indexing:
+  * Directly include predicate evaluation in ``STRtree.query()`` (#87)
+  * Query multiple input geometries (spatial join style) with ``STRtree.query_bulk`` (#108)
+* Fixed ``STRtree`` creation to allow querying the tree in a multi-threaded
+  context (#361).
+
+Bindings for new GEOS functionalities
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Several (new) functions from GEOS are now exposed in Shapely:
+
+* ``haussdorff_distance`` and ``frechet_distance()``
```

Comment:
We could turn this into links to the API docs (if we include this in the sphinx documentation site)